src/IntType.cc

```diff
@@ -98,7 +98,11 @@ IntType::IntType(JSContext *cx, JS::BigInt *bigint) {
   // Cast to a pythonmonkey.bigint to differentiate it from a normal Python int,
   //  allowing Py<->JS two-way BigInt conversion.
   // We don't do `Py_SET_TYPE` because `_PyLong_FromByteArray` may cache and reuse objects for small ints
+  #if PY_VERSION_HEX >= 0x03090000
   pyObject = PyObject_CallOneArg(PythonMonkey_BigInt, pyIntObj); // pyObject = pythonmonkey.bigint(pyIntObj)
+  #else
+  pyObject = PyObject_CallFunction(PythonMonkey_BigInt, "O", pyIntObj); // PyObject_CallOneArg is not available in Python < 3.9
```

Comment:
`_PyObject_CallOneArg` does not exist in Python 3.8, although mentioned in the [doc](https://docs.python.org/3.9/c-api/call.html?highlight=pyobject_callonearg#c.PY_VECTORCALL_ARGUMENTS_OFFSET)